yfinance/scrapers/history.py

```diff
@@ -98,6 +99,14 @@ def history(self, period="1mo", interval="1d",
             if start is None:
                 if interval == "1m":
                     start = end - 604800  # Subtract 7 days
+                
+                elif interval == "1h":
+                    start = end - 63072000 # Subtract 730 days
+                
+                # 5m to 90m interval supports up to 60 days
+                elif interval in ("5m", "15m", "30m", "60m", "90m"):
```

Comment:
I tested these intervals with the main branch, and they all support up to 60 days, including 60m interval which should be equivalent to 1h, unless there is another reason for it?

Reply:
Last I checked, Yahoo treats `1h` different, which could be different to `60m`.

Reply:
60m is equivalent to 1h, so I've fixed.